components/navbar/navbar.snip.html

```diff
@@ -19,10 +19,15 @@
 {{#navbar}}
   <!-- Start Navbar -->
   <header class="ampstart-headerbar fixed flex justify-start items-center top-0 left-0 right-0 pl2 pr4">
-    <div role="button" aria-label="open sidebar" on="tap:{{sidebar-id}}.toggle" tabindex="0" class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2">☰</div>
+    <div role="button" aria-label="open sidebar" on="tap:{{sidebar-id}}.toggle" tabindex="0" class="ampstart-navbar-trigger {{^sidebaronly}}md-hide lg-hide{{/sidebaronly}} pr2">
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
- this svg has a weird path when its focused we need to make it square  
- we use svgo to optimize
- can you put it in a separate file and inline it by including it here 
- also now that i think about this, changing/adjusting the image becomes very problematic with this approach. I think we should retain the css/unicode char and find a better UTF8 char. 
- If you still insist on the svg image make fixes 1 ,2,3